### PR TITLE
[IN-0] Host test refactoring

### PR DESCRIPTION
**Pull request by fehersanyi-bitrise**

### CODE REVIEW

File: tests/host_tests.py [hunk 1]
```diff
@@ -22,35 +20,26 @@ def test_datadog_agent_is_installed(host):
...
     host.file("$HOME/.ssh/id_rsa.pub").exists
 # Test if tools are installed
 def test_anka_is_available(host):
-    output = host.run("/usr/local/bin/anka version").rc
-    assert output == 0
+    assert host.run("/usr/local/bin/anka version").rc == 0
 def test_packer_is_available(host):
-    output = host.run("/usr/local/bin/packer version").rc
-    assert output == 0
+    assert = host.run("/usr/local/bin/packer version").rc == 0
```
Comment (MrBlaise) [inline tests/host_tests.py:37]: `=` after assert is not needed (same comment to all the lines below this)

Comment (fehersanyi-bitrise) [reply tests/host_tests.py:37]: you mean I should get rid of the `== 0` part because assert can evaluate the truthly value of the first argument?

Comment (MrBlaise) [reply tests/host_tests.py:37]: I mean the equal sign after assert `assert = host.run`
```suggestion
    assert host.run("/usr/local/bin/packer version").rc == 0
```

Comment (fehersanyi-bitrise) [reply tests/host_tests.py:37]: @MrBlaise

Comment (fehersanyi-bitrise) [reply tests/host_tests.py:37]: oooo I get it, those are copy pasta mistakes 🤦‍♂️

**Review by MrBlaise — APPROVED** [review]

**Review by peterheja-bitrise — APPROVED** [review]
LGTM

**Pull request merged**